devicon.json

```diff
@@ -4684,6 +4684,34 @@
         },
         "color": "#FDCC21",
         "aliases": []
+    },{
+        "name": "jetpackcompose", 
+
+        "altnames": ["jetpack","compose"], 
+
+        "tags": ["framework","language","kotlin","android"], 
+
+        "versions": {
+
+            "svg": ["original",
+            "original-wordmark",
+            "plain",
+            "plain-wordmark",
+            "line",
+            "line-wordmark"], 
+
+
+            "font":["original",
+            "original-wordmark",
+            "plain",
+            "plain-wordmark",
+            "line",
+            "line-wordmark"] 
+        },
+
+        "color": "#4285F4", 
+
+        "aliases": [] 
```

Comment:
Please use proper JSON indentation. You can take inspiration from other keys/icons. 🙂

Reply:
Updated devicon.json with change in indentation. 👍